ansible/group_vars/all/variables.yml

```diff
@@ -10,6 +10,10 @@ hashicorp:
     vault: 1.5.0
     nomad: 0.12.2
     packer: 1.6.1
+  tools_enterprise:
+    consul: 1.8.3+ent
+    nomad: 0.12.3+ent
+    vault: 1.5.0+ent
 ansible:
```

Comment:
What about terraform enterprise version?

Reply:
It is a cloud solution and does not apply ;)